### ISSUE TITLE
Admin triaging of new city applications.

### ISSUE THREAD

**Issue by asendecka** [DjangoGirls/djangogirls#342]
Ticket for: https://trello.com/c/ku0CFC3b/2-create-triaging-of-the-submitted-organization-requests

We need to implement the following (let's split it into smaller bits and pieces once we have a general structure in place)

- [x] Add `status` to Event application model with options: `NEW | IN REVIEW | ON_HOLD | ACCEPTED | REJECTED`, default: `NEW`
- [x] Add `rejection_reason` to Event application model, nullable, blankable
- [x] Add `comment` to Event application model, nullable, blankable
- [x] Add columns on event application Django admin page: `main organizer (email)` (with e-mail in parenthesis), `status` and `comment`
- [x] Expose `rejection_reason` and `comment` in Django admin
- [x] Add validation: if the status is changed to `On hold`, `comment` is required
- [x] Add validation: if the status is changed to `Rejected`, `rejection_reason` is required
- [x] @asendecka Move logic of deploy new city management command to the method on Event application model
- [ ] @asendecka Add logic for copy event from existing one if the `EventApplication` has `previous_event` set
- [x] @olasitarska Create a rejection method on Event application model
- [x] Add `Accept` & `Reject` buttons to edit form. Make confirmation page: "are you sure you want to Accept/Reject the page" and warn about clicking will send e-mails.
- [x] Add `On hold` and `In review` buttons that simply change status of the event application
- [ ] @asendecka Send e-mail to all organizers about deployed city (use Mandrill) - we will also need to integrate it with the work @helenst does to automate e-mail user creation (we cc djangogirls email for the city)
- [x] @olasitarska Send rejection e-mail to main organizer & other organizers provided in Event application (use Mandrill)
- [x] Add bulk actions on event application list page: for changing status to `On hold` or `In review`
- [x] ~~@olasitarska Embed `rejection reason` in the rejection e-mail~~
- [x] Add filter on top of list page by status (similar to the one on the event list page)
- [ ] Tests 💯

**Comment by asendecka** [comment]
We might split some of the above to separate tickets, so we can work in parallel after we have main things: status, accept/reject methods and new fields created.
